website/static/js/tests/registrationUtils.test.js

```diff
@@ -0,0 +1,399 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
```

Comment:
Because this file is under active revisions of of Wednesday evening 10/28, I'll give it only a light pass for the moment.